meta/regexp/character_types.rb

```diff
@@ -0,0 +1,111 @@
+Mutant::Meta::Example.add :regexp_word_type do
```

Comment:
This is very repetitive, should we instead do something like this:

```
[ 
  :regexp_word_type, '\w', '\W',
  ....
].each do |node_type, original, replacement|
  Mutant::Meta::Example.add(node_type) do
    source("/#{original}/")
    singleton_mutations
    mutation('//')
    mutation '/nomatch\A'
    mutation("/#{replacement}/")
end
```

Its far easier to review this way.